controllers/activemqartemis_reconciler.go

```diff
@@ -476,10 +479,20 @@ func (reconciler *ActiveMQArtemisReconcilerImpl) sourceEnvVarFromSecret(fcustomR
 				}
 			}
 		}
+		//if operator doesn't own it, don't track
+		if len(secretDefinition.OwnerReferences) > 0 {
+			for _, or := range secretDefinition.OwnerReferences {
+				if or.Kind == "ActiveMQArtemis" && or.Name == customResource.Name {
```

Comment:
Is that "ActiveMQArtemis" constant defined somewhere else that we can reference?

Reply:
Yeah it should, let me find.

Reply:
Didn't find existing one. I added a var to suite_test.go.